cluster/addons/ip-masq-agent/ip-masq-agent.yaml

```diff
@@ -21,6 +21,13 @@ spec:
       containers:
       - name: ip-masq-agent
         image: gcr.io/google-containers/ip-masq-agent-amd64:v2.0.1
+        resources:
+          limits:
+            cpu: 10m
```

Comment:
One question here: How about not adding `request` here? Sometimes the pod may not able to start due to not enough resources, not adding `request` can make the pod can start up.